CHANGES.md

```diff
@@ -24,6 +24,11 @@ OpenSSL 3.1
 
 ### Changes between 3.0 and 3.1 [xx XXX xxxx]
 
+ * CCM8 cipher suites in TLS have been downgraded to security level 1 because
+   they use a short tag which lowers their strength.
```

Comment:
I forget if we like "tag" vs "authentication tag"

Reply:
I doubt it matters but I'll change to use _authentication tag_.